scalalib/src/ScalaModule.scala

```diff
@@ -283,7 +302,10 @@ trait ScalaModule extends JavaModule { outer =>
 
       packageWithZinc(
         Seq("-d", javadocDir.toNIO.toString),
-        allSourceFiles().map(_.path.toString),
+        docSources()
+          .map(_.path)
+          .filter(_.ext == "tasty")
```

Comment:
The tasty file filter is wrong for Scala 2, instead it should select same files as `allSourceFiles` does.

Reply:
My bad. I read "Scala 3" instead of "2" on the above comment.

Fixed in https://github.com/com-lihaoyi/mill/pull/1385/commits/c214aa7244d8d4d719c7ef4afaaf99c39bcabdd4